internal/resources/package/data_source_package.go

```diff
@@ -40,3 +54,116 @@ var packageSchema = map[string]*schema.Schema{
 	common.MetaKey:       common.Meta,
 	spec.SpecKey:         spec.SpecSchema,
 }
+
+func resourcePackageRead(ctx context.Context, d *schema.ResourceData, m interface{}) (diags diag.Diagnostics) {
+	config := m.(authctx.TanzuContext)
+
+	packageName, ok := d.Get(nameKey).(string)
+	if !ok {
+		return diag.Errorf("unable to read package name")
+	}
+
+	packageNamespaceName, ok := d.Get(namespaceKey).(string)
+	if !ok {
+		return diag.Errorf("Unable to read package namespace name")
```

Comment:
nit: maintain consistency - either start the err message in upper or lower case as per the how the existing resources follow

Reply:
updated all messages to start from upper case